src/components/CardList.js

```diff
@@ -0,0 +1,78 @@
+import "../assets/styles/CardList.css";
+import logo from "../assets/images/logo.png";
+
+const formatDate = (value) => {
+  const date = new Date(value);
+  return `${date.getFullYear()}. ${date.getMonth() + 1}. ${date.getDate()}`;
+};
+
+const caculateTime = (value) => {
+  const createdDate = new Date(value);
+  const nowDate = new Date();
+  const timeDiff = nowDate - createdDate;
+
+  const minDiff = Math.floor(timeDiff / 1000 / 60);
+  const hourDiff = Math.floor(minDiff / 60);
+  const dayDiff = Math.floor(hourDiff / 24);
+  const monthDiff = Math.floor(dayDiff / 31);
+  if (monthDiff >= 12) {
+    return `${Math.floor(monthDiff / 12)} years ago`;
+  } else if (monthDiff >= 1) {
+    return `${monthDiff} months ago`;
+  } else if (dayDiff >= 1) {
+    return `${dayDiff} days ago`;
+  } else if (hourDiff >= 1) {
+    return `${hourDiff} hours ago`;
+  } else if (minDiff >= 2) {
+    return `${minDiff} minutes ago`;
+  } else {
+    return "1 minute ago";
+  }
```

Comment:
```suggestion
  if (monthDiff >= 12) {
    return `${Math.floor(monthDiff / 12)} years ago`;
  }
  if (monthDiff >= 1) {
    return `${monthDiff} months ago`;
  } 
  if (dayDiff >= 1) {
    return `${dayDiff} days ago`;
  } 
  if (hourDiff >= 1) {
    return `${hourDiff} hours ago`;
  } 
  if (minDiff >= 2) {
    return `${minDiff} minutes ago`;
  } 
   return "1 minute ago";
  
```

로직 자체가 복잡해서 더 깔끔하게 짜긴 어려울것 같습니다
다만 if문에서 return이 된다면 굳이 else if를 사용할 필요가 없어서 위처럼 작성하는걸 추천드립니다!